python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -146,6 +176,212 @@ def create_domain(
 
         return domain_urn
 
+    def upsert_athena_database(self, metadata: DatabaseMetadata) -> str:
+        """
+        Define a database. Must belong to a domain
+        """
+        metadata_dict = dict(metadata.__dict__)
+        metadata_dict.pop("version")
```

Comment:
why are you popping these?

Reply:
Similar to the data product upsert methods these are popped because they're not used. One for the wider review of metadata entities mentioned in the PR comment i think